catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -94,57 +109,38 @@ public void refreshRegistrySubscriptions() throws FederationAdminException {
 
-            return registryMetacards.stream()
-                    .filter(RegistryUtility::isRegistryMetacard)
-                    .collect(Collectors.toMap(RegistryUtility::getRegistryId, Function.identity()));
+            return registryMetacards.stream().collect(Collectors
+                    .toMap(e -> e.getAttribute(RegistryObjectMetacardType.REMOTE_METACARD_ID)
+                            .getValue().toString(), Function.identity()));
         } catch (Exception e) {
             throw new FederationAdminException("Error querying for metacards ", e);
         }
     }
 
-    private Map<String, Metacard> getRemoteRegistryMetacardsMap() throws FederationAdminException {
+    private RemoteRegistryResults getRemoteRegistryMetacardsMap() {
         Map<String, Metacard> remoteRegistryMetacards = new HashMap<>();
-        Set<String> sourceIds = registryStores.stream()
-                .filter(RegistryStore::isAvailable)
-                .filter(RegistryStore::isPullAllowed)
-                .map(RegistryStore::getId)
-                .collect(Collectors.toSet());
-
-        if (CollectionUtils.isEmpty(sourceIds)) {
-            return remoteRegistryMetacards;
-        }
-
-        try {
-
-            List<Metacard> fullList =
-                    Security.runAsAdminWithException(() -> federationAdminService.getRegistryMetacards(
-                            sourceIds));
-            for (Metacard metacard : fullList) {
-                String registryId = RegistryUtility.getRegistryId(metacard);
-                if(registryId == null){
-                    continue;
-                }
-                if (remoteRegistryMetacards.containsKey(registryId)) {
-                    Metacard currentMetacard = remoteRegistryMetacards.get(registryId);
-                    if (currentMetacard.getModifiedDate()
-                            .before(metacard.getModifiedDate())) {
-                        remoteRegistryMetacards.put(registryId, metacard);
-                    }
-                } else {
-                    remoteRegistryMetacards.put(registryId, metacard);
-                }
+        List<String> failedQueries = new ArrayList<>();
+        SourceResponse response;
+        for (RegistryStore store : registryStores) {
+            if (!store.isPullAllowed() || !store.isAvailable()) {
+                continue;
+            }
+            try {
+                response = store.query(new QueryRequestImpl(getBasicRegistryQuery()));
+                remoteRegistryMetacards
+                        .putAll(response.getResults().stream().map(e -> e.getMetacard())
+                                .collect(Collectors.toMap(Metacard::getId, Function.identity())));
+            } catch (UnsupportedQueryException e) {
+                LOGGER.warn("Unable to contact {} for registry subscription query", store.getId());
+                LOGGER.debug("Detailed exception:", e);
```

Comment:
Can't log just the debug but made it log the same message as the warn log statement